arches_component_lab/views/api/widgets.py

```diff
@@ -26,18 +27,32 @@ def update_i18n_properties(response):
 
 class WidgetDataView(View):
     def get(self, request, graph_slug, node_alias):
-        card_x_node_x_widget = models.CardXNodeXWidget.objects.filter(
-            node__graph__slug=graph_slug,
-            node__alias=node_alias,
-            node__source_identifier_id__isnull=True,
-        ).first()
+        card_x_node_x_widget = (
+            models.CardXNodeXWidget.objects.filter(
+                node__graph__slug=graph_slug,
+                node__alias=node_alias,
+                node__source_identifier_id__isnull=True,
```

Comment:
This isn't going to work on 7.6, will need a version switch here for building the filter args